packages/lib/passport-interface/src/CheckinCredential.ts

```diff
@@ -0,0 +1,78 @@
+import { EmailPCD, EmailPCDPackage } from "@pcd/email-pcd";
+import { SerializedPCD } from "@pcd/pcd-types";
+import {
+  SemaphoreSignaturePCD,
+  SemaphoreSignaturePCDPackage
+} from "@pcd/semaphore-signature-pcd";
+
+/*
+ * The payload encoded in the message of the SemaphoreSignaturePCD passed
+ * as a credential to the checkin api within the Generic Issuance backend.
+ */
+export interface GenericCheckinCredentialPayload {
+  emailPCD: SerializedPCD<EmailPCD>;
+  ticketIdToCheckIn: string;
+  eventId: string;
+  timestamp: number;
+}
+
+/**
+ * After verification, return the unserialized Email PCD.
+ */
+export interface VerifiedCheckinCredential {
+  emailPCD: EmailPCD;
+  ticketIdToCheckIn: string;
+  eventId: string;
+}
+
+/**
+ * Creates a credential payload for use in the Generic Issuance checkin API.
+ */
+export function createGenericCheckinCredentialPayload(
```

Comment:
i wonder if any of these helper functions could be used in `genericIssuance.spec.ts`

Reply:
This one already is. There's one more helper function that I ought to move from `genericIssuance.spec.ts` to `passport-interface` - I'll do that tomorrow.